index.html

```diff
@@ -209,14 +217,14 @@ <h4>Skills</h4>
     <footer>
         <div class="footer-info">
             <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
-                <form action="contact">
-                    <input type="text" name="name" placeholder="Full name" required/>
-                    <input type="text" name="email" placeholder="Email address" required/>
-                    <textarea name="comment" id="comment" cols="30" rows="5">Enter your text here</textarea>
-                    <div class="btn-center">
-                        <button class="btn btn-style submit-form" type="button" >Get in touch</button>
-                    </div>
-                </form>
+            <form action="https://formspree.io/f/form_id" method="post">
```

Comment:
- [ ] You did a great job implementing required changes but as previous reviewer kindly remove form_id and put the actual value generated [formspree](https://formspree.io/) when signed up. Keep rocking 💪